activesupport/test/caching_test.rb

```diff
@@ -809,11 +809,6 @@ def test_clear_without_cache_dir
     @cache.clear
   end
 
-  def test_long_keys
-    @cache.write("a"*10000, 1)
```

Comment:
Since this is being deleted we should change the tests that is duplicated to have the same key size of this one. Right now it is only 900 characters.